### PR TITLE
WIP - REPL: bring old console.log pane back [#1392]

### PR DESCRIPTION
**Pull request by rajasekarm**
- [ ]  Design review from @existentialism

### CODE REVIEW

**Comment by bvaughn** [comment]
FYI layout looks a little off on mobile:
screen shot 2017-11-14 at 1 58 08 pm

File: js/repl/CodeMirrorPanel.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { css } from "emotion";
 import CodeMirror from "./CodeMirror";
 import React from "react";
 import { colors } from "./styles";
+import prettyFormat from "pretty-format";
```
Comment (hzoo) [inline js/repl/CodeMirrorPanel.js]: Should we be using this or https://github.com/pretty-format/pretty-format2 @thejameskyle

Comment (bvaughn) [reply js/repl/CodeMirrorPanel.js]: My preference would be neither, unless we're loading from a CDN. This package adds 16.1kB gzipped to the bundle.

(`pretty-format2` would only add 8kB gzipped, but still)

Comment (hzoo) [reply js/repl/CodeMirrorPanel.js]: we can use a cdn for sure, otherwise we run it through json.stringify

**Comment by bvaughn** [comment]
+1 for stringify. KISS.

**Comment by rajzshkr** [comment]
Updated the PR with JSON.stringify. 

@bvaughn - Log panel will be shown for the compiled output side.

**Comment by bvaughn** [comment]
> @bvaughn - Log panel will be shown for the compiled output side.

The layout is still broken for mobile sized screens.

screen shot 2017-11-16 at 8 27 35 am

**Comment by NE-SmallTown** [comment]
Is there any update on this? :)

**Comment by rajzshkr** [comment]
@NE-SmallTown  I've to fix a small alignment issue. Will ship this weekend.

**Pull request closed without merging**